pywr/parameters/_parameters.pyx

```diff
@@ -290,12 +290,15 @@ cdef class TablesArrayParameter(IndexParameter):
         self.where = where
         self.scenario = scenario
 
-        # Private attributes, initialised during reset()
+        # Private attributes, initialised during setup()
         self._values_dbl = None
         self._values_int = None
         self._scenario_index = -1
+        self._scenario_ids = None
```

Comment:
See above. Would be good to summarise in a comment what these are all used for in the upcoming code.